build.sh

```diff
@@ -16,6 +16,10 @@ fi
 export GO15VENDOREXPERIMENT=1
 export GOPATH=${PWD}/gopath
 
+if [ -z ${GO+x} ]; then
```

Comment:
Is there any reason not to just do `GO="${GO:-go}"` ?

Reply:
sure, i'll give it a try, this was just a dumb noobie way to get it going I guess :\